dashboard/app/controllers/foorm/forms_controller.rb

```diff
@@ -0,0 +1,43 @@
+module Foorm
+  class FormsController < ApplicationController
+    before_action :require_levelbuilder_mode_or_test_env
+    before_action :authenticate_user!
+    authorize_resource
+
+    # PUT foorm/form/:name/:version
+    def update
+      form_questions = JSON.pretty_generate(params[:questions].as_json)
```

Comment:
Does `pretty_generate` require JSON as input?  I thought it make take `params[:questions]` as is, without requiring the `.as_json`...

Reply:
it requires a 'JSON object or array', and fails if you don't do `as_json` because `params[:questions]` is  `ActionController: Parameters`